test/upstream-requirements-py37.txt

```diff
@@ -1,26 +1,25 @@
-attrs==21.2.0
```

Comment:
@texastony, how come we are changing this file? are we just refreshing the deps here?

from the conversation over in https://github.com/pyca/cryptography/pull/8115#issuecomment-1400769056
we are doing both of their suggestions. Is there a need for both or can we just update this file?

Reply:
Actually, I am some-what interested in the diff of these two files.
But yes, by updating py37.txt, we are updating the dependencies frozen there to those used when testing with python 3.7.12.

Reply:
I am not against dropping 3.7 entirely, but I think that Pyca depends on it...
This way, we offer them both, and if they ever tell us we only need 311, we can drop 3.7.

@alex , can you let us know if you no longer need 3.7?

Reply:
We don't need py37 anymore -- we only test our downstreams against a single python version.